src/main/java/uk/gov/companieshouse/api/accounts/MongoConfig.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.companieshouse.api.accounts;
+
+import org.springframework.beans.factory.BeanFactory;
+import org.springframework.beans.factory.NoSuchBeanDefinitionException;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.mongodb.MongoDbFactory;
+import org.springframework.data.mongodb.core.convert.CustomConversions;
+import org.springframework.data.mongodb.core.convert.DbRefResolver;
+import org.springframework.data.mongodb.core.convert.DefaultDbRefResolver;
+import org.springframework.data.mongodb.core.convert.DefaultMongoTypeMapper;
+import org.springframework.data.mongodb.core.convert.MappingMongoConverter;
+import org.springframework.data.mongodb.core.mapping.MongoMappingContext;
+
+/**
+ * @author dparameswaran
+ * _class maps to the model class in mongoDB (i.e. _class : uk.gov.companieshouse.Transaction)
+ * when using spring data mongo it by default adds a _class key to your collection to be able to
+ * handle inheritance. But if your domain model is simple and flat, you can remove it by overriding
+ * the default MappingMongoConverter
+ *
+ */
+@Configuration
+@SuppressWarnings("unused")
+public class MongoConfig {
+
+    @Bean
+    public MappingMongoConverter mappingMongoConverter(MongoDbFactory factory, MongoMappingContext context,
+            BeanFactory beanFactory) {
+        DbRefResolver dbRefResolver = new DefaultDbRefResolver(factory);
+        MappingMongoConverter mappingConverter = new MappingMongoConverter(dbRefResolver, context);
+        try {
+            mappingConverter.setCustomConversions(beanFactory.getBean(CustomConversions.class));
+        } catch (NoSuchBeanDefinitionException ignore) {
+            // will need to output error
```

Comment:
It would be better to add the logging than comments. This might get missed later.

Reply:
done